zuul.d/jobs.yaml

```diff
@@ -90,6 +90,16 @@
       - tools/manage-projects.py
       - tox.ini
 
+- job:
+    name: project-config-tox-linters
+    parent: ansible-tox-linters
+    description: |
+      Run some policy checks on zuul.d/projects.yaml.
+      Uses ``linters`` tox env.
```

Comment:
Since we write the secret directly on the disk, you need `final: true` to be sure nobody will prepare a job in `ansible-zuul-jobs` that inherit from it.

Reply:
Done. Thanks